libraries/core-react/CHANGELOG.md

```diff
@@ -5,6 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.2] - 2020-09-15
+
+### Fixed 🐛
+
+- Fixed `onClose` handler not being called when `MenuItem` was used inside `MenuSection` ([#546](https://github.com/equinor/design-system/issues/546))
+- Fixed focus showing when clicking on `MenuItem` ([#544](https://github.com/equinor/design-system/issues/544))
+
+### Changed
+
+- Added outside click support for closing `Menu`. Outside clicks will now call the `onClose` handler function. ([#548](https://github.com/equinor/design-system/issues/548))
+- Added `data` property to `Icon` component to easily compose icon to be rendered. ([#584](https://github.com/equinor/design-system/issues/584))
+
+```jsx
+import { save } from '@equinor/eds-icons';
+
+// EDS icon
+<Icon data={save} />
+
+// Custom icon
+<Icon data={{
+  prefix: 'custom',
+  height: '24',
+  width: '24',
+  svgPathData: 'M12 16.067l4.947 3.6-1.894-5.814L20 10.334h-6.067l-1.933-6-1.933 6H4l4.947 3.52-1.894 5.814 4.947-3.6z',
+}}>
+```
```

Comment:
Don’t include code examples in the changelog, better to link to the storybook

Reply:
@vnys Storybook does not show import statements unfortunately, it only shows the content on the render function you export 🤔

Reply:
A README file somewhere then?

Reply:
@vnys Good suggestion, linked to README file for `Icon` component instead